src/app/plugins/addclasschans.plugin.ts

```diff
@@ -140,10 +140,15 @@ export default class AddClassChannelsPlugin extends Plugin {
   }
 
   private _createFirstMessage(chanName: string): MessageEmbed {
+    const invChan = this.container.guildService
+      .get()
+      .channels.cache.find((c) => c.name === Constants.Channels.Info.ClassInvite);
```

Comment:
We probably have to fetch. I doubt the channel will be in the cache because the class invite channel is stale 99% of the time